### DISCUSSION TITLE
react-page/editor doesn't work correctly

### DISCUSSION THREAD

**Discussion by katerynapadafa** [react-page/react-page#1309] [Q&A] [answered]
Hi there. I need your help. The styles you applied to UI don''t work. We don't use MUI to style components in our project. The same issue with resize button and move button. Could you tell me how to fix that? The demo version looks nice but in real project it looks like a raw version.

**Comment by macrozone** [comment, marked as answer]
do you import the css file as well? https://react-page.github.io/docs/#/quick-start?id=dependencies

**Reply by katerynapadafa** [reply]
i didn't 
thank you :)
